internal/acceptance/lakeview_dashboard_test.go

```diff
@@ -0,0 +1,477 @@
+package acceptance
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/databricks/terraform-provider-databricks/qa"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type templateStruct struct {
+	DisplayName         string
+	WarehouseId         string
+	ParentPath          string
+	SerializedDashboard string
+	FilePath            string
+	EmbedCredentials    string
+}
+
+func makeTemplate(template templateStruct) string {
+	templateString := fmt.Sprintf(`
+	resource "databricks_lakeview_dashboard" "d1" {
+		display_name			= 	"%s"
+		warehouse_id			=	"%s"
+		parent_path				= 	"%s"
+	`, template.DisplayName, template.WarehouseId, template.ParentPath)
+	if template.SerializedDashboard != "" {
+		templateString += fmt.Sprintf(`	serialized_dashboard	=	"%s"
+	`, template.SerializedDashboard)
+	}
+	if template.FilePath != "" {
+		templateString += fmt.Sprintf(`	file_path				=	"%s"
+	`, template.FilePath)
+	}
+	if template.EmbedCredentials != "" {
```

Comment:
Not sure if spacing is accidental indentation issue.

Reply:
It is intentional (just to make the request template in debug log well formatted)